docs/pages/versions/unversioned/sdk/audio-av.mdx

```diff
@@ -0,0 +1,197 @@
+        /* @end */ Audio.RecordingOptionsPresets.HIGH_QUALITY
+      );
+      setRecording(recording);
+      console.log('Recording started');
+    } catch (err) {
+      console.error('Failed to start recording', err);
+    }
+  }
+
+  async function stopRecording() {
+    console.log('Stopping recording..');
+    setRecording(undefined);
+    /* @info */ await recording.stopAndUnloadAsync(); /* @end */
+    /* @info iOS may reroute audio playback to the phone earpiece when recording is allowed, so disable once finished. */ await Audio.setAudioModeAsync(
+      {
+        allowsRecordingIOS: false,
+      }
+    ); /* @end */
+    /* @info */ const uri = recording.getURI(); /* @end */
+    console.log('Recording stopped and stored at', uri);
+  }
+
+  return (
+    <View style={styles.container}>
+      <Button
+        title={recording ? 'Stop Recording' : 'Start Recording'}
+        onPress={recording ? stopRecording : startRecording}
+      />
+    </View>
+  );
+}
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    justifyContent: 'center',
+    backgroundColor: '#ecf0f1',
+    padding: 10,
+  },
+});
+```
+
+</SnackInline>
+
+### Playing or recording audio in background&ensp;<PlatformTags platforms={['ios']} />
+
+On iOS, audio playback and recording in background is only available in standalone apps, and it requires some extra configuration.
+On iOS, each background feature requires a special key in `UIBackgroundModes` array in your **Info.plist** file.
+In standalone apps this array is empty by default, so to use background features you will need to add appropriate keys to your **app.json** configuration.
+
+See an example of **app.json** that enables audio playback in background:
+
+```json
+{
+  "expo": {
+    ...
+    "ios": {
+      ...
+      "infoPlist": {
+        ...
+        "UIBackgroundModes": [
+          "audio"
+        ]
+      }
+    }
+  }
+}
+```
+
+### Notes on web usage
+
+- A MediaRecorder issue on Chrome produces WebM files missing the duration metadata. [See the open Chromium issue](https://bugs.chromium.org/p/chromium/issues/detail?id=642012).
+- MediaRecorder encoding options and other configurations are inconsistent across browsers, utilizing a Polyfill such as [kbumsik/opus-media-recorder](https://github.com/kbumsik/opus-media-recorder) or [ai/audio-recorder-polyfill](https://github.com/ai/audio-recorder-polyfill) in your application will improve your experience. Any options passed to `prepareToRecordAsync` will be passed directly to the MediaRecorder API and as such the polyfill.
+- Web browsers require sites to be served securely for them to listen to a mic. See [MediaDevices `getUserMedia()` security](https://developer.mozilla.org/en-US/docs/Web/API/MediaDevices/getUserMedia#security) for more details.
+
+## API
+
+```js
+import { Audio } from 'expo-av';
+```
+
+<APISection packageName="expo-audio" apiName="Audio" />
```

Comment:
```suggestion
<APISection packageName="expo-audio-av" apiName="Audio" />
```

To fetch the correct reference as per the change in `GenerateDocsAPIData.ts`, lets update the package name here.